crates/bcr-wdc-webapi/src/quotes.rs

```diff
@@ -123,6 +123,24 @@ pub enum StatusReply {
 }
 
 /// --------------------------- List quotes
+#[derive(Serialize, Deserialize, ToSchema)]
+pub struct ListFilterParam {
+    pub bill_maturity_date_from: Option<chrono::NaiveDate>,
+    pub bill_maturity_date_to: Option<chrono::NaiveDate>,
+    pub status: Option<StatusReplyDiscriminants>,
+    pub bill_drawee_id: Option<String>,
```

Comment:
in my understanding (at least on ebill), the drawee is the payer, so `bill_drawee_id` and `bill_payer_id` would always be the same.